tests/dragonfly/replication_test.py

```diff
@@ -2365,7 +2365,7 @@ async def test_replication_timeout_on_full_sync(df_factory: DflyInstanceFactory,
     seeder.stop()
     await seeder_task
 
-    await check_all_replicas_finished([c_replica], c_master, timeout=30)
+    await check_all_replicas_finished([c_replica], c_master, timeout=60)
```

Comment:
I am hesitant that this fixes anytthing. Locally with 2 proactors it takes less than a second to replicate a master of this size so I don't see how increasing the timeout will help. The problem seems to be that replication is not making progress.

Reply:
I just run this test locally with 2 proactors and it took 11s to get to fullsync
and we also see in the logs that it takes 30s to get to fullsync in the github machine